recipes/hitac/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set version = "1.0" %}
+
+package:
+  name: hitac
+  version: {{ version }}
+
+build:
+  number: 0
+  noarch: python
+source:
+  url: https://gitlab.com/dacs-hpi/hitac/uploads/ea79d6d24da7b72038199f2cccebf724/hitac
```

Comment:
You should use this URL: `https://gitlab.com/dacs-hpi/hitac/-/archive/v{{ version }}/hitac-v{{ version }}.tar.gz`.

In this way, the URL can be easily updated on the next release (possibly even automatically by the BiocondaBot, not sure if it works with GitLab).

Reply:
I updated to https://gitlab.com/dacs-hpi/hitac/uploads/451db23c35b8b03f282fcf34b61b3b93/hitac-{{ version }}.tar.gz
This was the best I could do with gitlab

Reply:
I think something is wrong with the release that you tagged. When I go to https://gitlab.com/dacs-hpi/hitac/-/releases/ , I can see the one release that you made and it is called "v1.1", but the tag (on the same line as "Created x hours ago" says that it is "v1.0", and also clicking on the "source code" dropdown gives me a list of links that all contain "v1.0" in the URL.

I don’t know how making releases on GitLab works, but it appears that something went wrong this time.

Reply:
Ok, I think I know what you did: Possibly you made a release by tagging v1.0, but then you changed the title of the release to v1.1. You should change the name back to v1.0 and then tag a new release v1.1.

Reply:
> Ok, I think I know what you did: Possibly you made a release by tagging v1.0, but then you changed the title of the release to v1.1. You should change the name back to v1.0 (to reduce confusion) and then tag a new release v1.1.

Oops. You are right. I reverted the name and tagged a new release. I also updated the URL and sha256 on the meta.yaml file.

Reply:
Hello @marcelm,

I believe now my recipe is complying with your requests. Can you please review again or let me know if I missed something?

Regards